doc/source/changes/version_0_30.rst.inc

```diff
@@ -13,6 +13,21 @@ Backward incompatible changes
 New features
 ------------
 
+* added LArray.isin method to check whether each element of an array is contained in a list (or array) of values.
+
+    >>> arr = ndtest((2, 3))
+    >>> arr
+    a\b  b0  b1  b2
+     a0   0   1   2
+     a1   3   4   5
+    >>> arr.isin([1, 2, 3, 5, 7])
+    a\b     b0     b1    b2
+     a0  False   True  True
+     a1   True  False  True
+    >>> arr[arr.isin([1, 5, 9])]
```

Comment:
I think you should use the same list of values in both examples otherwise people may be confused